Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "SCIP"
+uuid = "82193955-e24f-5292-bf16-6f2c5261a85f"
+
```

Comment:
I think the version is always required and must corresponds to the version yet to be released (not released yet)

Reply:
I see. And dropping `REQUIRE` would be considered a breaking change, I guess.

Reply:
yes I'd say so. Also, beware of `>=` compat, it means no upper-bound, which is problematic for breaking changes expected (MOI, CEnum next releases could break everything)